graphblas_algorithms/interface.py

```diff
@@ -82,6 +82,14 @@ def convert(graph, weight=None):
             return Graph.from_networkx(graph, weight=weight)
         raise TypeError(f"Unsupported type of graph: {type(graph)}")
 
+    @staticmethod
+    def convert_to_nx(obj, *, name):
```

Comment:
I think we should have `name=None`. I expect libraries to always provide `name`, but I think users should be encouraged to use this function, and we should be able to handle when `name` is None _most_ of the time, since the type of the argument is _probably_ all that is _usually_ needed.

`convert_from_nx` above should also have `name=None`.

Reply:
Done